integtest/src/main/java/org/corfudb/universe/node/CorfuServer.java

```diff
@@ -0,0 +1,46 @@
+package org.corfudb.universe.node;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import org.slf4j.event.Level;
+
+/**
+ * Represent a Corfu server implementation of {@link Node} used in the {@link org.corfudb.universe.cluster.Cluster}.
+ */
+public interface CorfuServer extends Node {
+
+    @Override
+    CorfuServer deploy() throws NodeException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.universe.node.NodeException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)